Dockerfile

```diff
@@ -49,8 +49,7 @@ RUN \
 # Cleanup
 RUN \
   apt-get clean && \
-  rm -fr /var/lib/apt/lists/* && \
-  rm -fr /tmp/*
```

Comment:
@Vbhatia20 can answer why this needed to deleted.. but without this it was not working.